### PR TITLE
M3808 As developer I want to stabilize the selenium test suite so that I can trust the results

### PR DESCRIPTION
**Pull request by fdlk**

### CODE REVIEW

File: src/main/java/org/molgenis/selenium/model/component/SpinnerModel.java [hunk 1]
```diff
@@ -0,0 +1,144 @@
+package org.molgenis.selenium.model.component;
+
+import static java.util.Arrays.asList;
+import static org.molgenis.selenium.test.AbstractSeleniumTest.IMPLICIT_WAIT_SECONDS;
```
Comment (bartcharbon) [inline src/main/java/org/molgenis/selenium/model/component/SpinnerModel.java]: This import fails, it seems not to like importing from the test classes

File: src/main/java/org/molgenis/selenium/util/RestApiV1Util.java [hunk 2]
```diff
@@ -2,7 +2,10 @@
 
 import static java.util.Arrays.asList;
 
+import org.apache.http.client.HttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
```
Comment (bartcharbon) [inline src/main/java/org/molgenis/selenium/util/RestApiV1Util.java]: cannot resolve those imports, maven module missing from the pom?

File: src/main/java/org/molgenis/selenium/model/component/SpinnerModel.java (hunk 1 above)
Comment (fdlk) [inline src/main/java/org/molgenis/selenium/model/component/SpinnerModel.java]: Interesting, will move to src dir

File: src/main/java/org/molgenis/selenium/util/RestApiV1Util.java (hunk 2 above)
Comment (fdlk) [inline src/main/java/org/molgenis/selenium/util/RestApiV1Util.java]: Same/similar issue: it had test scope. Works in eclipse, oddly enough.

**Comment by bartcharbon** [comment]
I got two failing tests:**testMapLifelinesToHop**(org.molgenis.selenium.test.mappingservice.MappingProjectDetailsTest)  Time elapsed: 17.551 sec  <<< FAILURE!
org.openqa.selenium.TimeoutException: Timed out after 10 seconds waiting for text ('$('WEIGHT').div(1000.0).div($('HEIGHT').div(100.0).pow(2)).value()')

tested with the resources from the download server:
**testAnnotateWithCaddAndSnpEffCopyDelete**(org.molgenis.selenium.test.dataexplorer.AnnotatorTest)  Time elapsed: 37.488 sec  <<< FAILURE!
java.lang.AssertionError: Lists differ at element [3]: [edit, trash, search, 3.852093, 23.4, missense_variant, MODERATE, ABCA4, ABCA4, transcript, NM_000350.2, Coding, 33/50, c.4685T>C, p.Ile1562Thr, 4789/7325, 4685/6822, 1562/2273, , , , ] != [edit, trash, search, , , missense_variant, MODERATE, ABCA4, ABCA4, transcript, NM_000350.2, Coding, 33/50, c.4685T>C, p.Ile1562Thr, 4789/7325, 4685/6822, 1562/2273, , , , ] expected:<[edit, trash, search, 3.852093, 23.4, missense_variant, MODERATE, ABCA4, ABCA4, transcript, NM_000350.2, Coding, 33/50, c.4685T>C, p.Ile1562Thr, 4789/7325, 4685/6822, 1562/2273, , , , ]> but was:<[edit, trash, search, , , missense_variant, MODERATE, ABCA4, ABCA4, transcript, NM_000350.2, Coding, 33/50, c.4685T>C, p.Ile1562Thr, 4789/7325, 4685/6822, 1562/2273, , , , ]>

**Comment by bartcharbon** [comment]
Seems that the expected CADD score is missing in the case of the annotator test

**Comment by fdlk** [comment]
Found out a bit of a quirk with the annotation settings: the default value for cadd is to use /srv/molgenis/.molgenis/omx/data/annotation_resources/CADD/1000G.vcf.gz and the annotator_settings on molgenis26 use /srv/molgenis/.molgenis/omx/annotation_resources/cadd/cadd.tsv.gz
That's not even in the same directory(!)

**Pull request merged**